tests/DoctrineORMModuleTest/Yuml/YumlControllerFactoryTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace DoctrineORMModuleTest\Yuml;
+
+use DoctrineORMModule\Yuml\YumlController;
+use DoctrineORMModule\Yuml\YumlControllerFactory;
+use Zend\ServiceManager\AbstractPluginManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class YumlControllerFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreateService()
+    {
+        $config = [
+            'zenddevelopertools' => [
+                'toolbar' => [
+                    'enabled' => true
+                ]
+            ]
+        ];
+
+        $serviceLocator = $this->getMockBuilder(ServiceLocatorInterface::class)
+            ->getMock();
+        $pluginManager = $this->getMockBuilder(AbstractPluginManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $serviceLocator->expects(static::any())->method('get')->with('config')->willReturn($config);
+
+        $pluginManager->expects(static::once())
+            ->method('getServiceLocator')
+            ->willReturn($serviceLocator);
+
+        $factory = new YumlControllerFactory();
+        $controller = $factory->createService($pluginManager);
+
+        static::assertInstanceOf(YumlController::class, $controller);
```

Comment:
You are completely right, I know the method is `static` and could be used with `static::` but author of the PHPUnit suggest to use it `$this->` and you can check all examples in PHPUnit docs.
Also in this library all other tests are with `$this->` and greater majority of all other php libraries on github! So maybe everyone is wrong, but you can notice how many people who are using PHPStorm notice it, and finally it's gonna be changed in PHPStorm (please read blog post I put above).

I'd like just have CONSISTENT code in the library. :)